Apps/WebClient/src/ClientApp/src/models/delegateInvitation.ts

```diff
@@ -0,0 +1,43 @@
+import { DataSource } from "@/constants/dataSource";
+
+/**
+ * Represents the delegate invitation model
+ */
+export interface DelegateInvitation {
```

Comment:
After chatting with Brian about the back-end implementation, we thought it might be best to split up the models so that CreateDelegateInvitationRequest will be different from the eventual UpdateDelegateInvitationRequest and returned DelegateInvitation.